apps/studio/src/server/modules/user/__tests__/user.service.test.ts

```diff
@@ -160,6 +187,41 @@ describe("user.service", () => {
           role: roleToCreate,
         }),
       ])
+
+      // Assert DB - audit logs (user)
+      const userAuditLogs = await db
+        .selectFrom("AuditLog")
+        .where("eventType", "=", "UserCreate")
+        .selectAll()
+        .execute()
+      expect(userAuditLogs).toHaveLength(1)
+      expect(userAuditLogs[0]).toMatchObject({
+        eventType: "UserCreate",
+        delta: expect.objectContaining({
+          before: null,
+          after: expect.objectContaining({
+            id: createdUser.id,
+            email: TEST_EMAIL,
+          }),
+        }),
+      })
```

Comment:
question - because we log the db object directly, do you think it's useful to query the object then match it here?

Reply:
not strongly opinionated here, willing to accept both. leaning towards db object because it's easier to read and this one requires understanding that `before/after` has this specific shape